### PR TITLE
chore(deps): update to std@0.189.0

### PR DESCRIPTION
**Pull request by iuioiua**

### CODE REVIEW

**Review by lino-levan — APPROVED** [review]
LGTM. Was working on a broader upgrade of fresh deps but as always, upgrading `esbuild`/`esbuild_deno_loader`/`preact`/`preact-render-to-string` causes things to explode. Will probably submit that PR soon.

File: src/dev/imports.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ export const RECOMMENDED_PREACT_RTS_VERSION = "5.2.6";
 export const RECOMMENDED_PREACT_SIGNALS_VERSION = "1.1.3";
 export const RECOMMENDED_PREACT_SIGNALS_CORE_VERSION = "1.2.3";
 export const RECOMMENDED_TWIND_VERSION = "0.16.19";
-export const RECOMMENDED_STD_VERSION = "0.187.0";
+export const RECOMMENDED_STD_VERSION = "0.188.0";
```
Comment (kt3k) [inline src/dev/imports.ts]: Should this also be 0.189.0?

Comment (iuioiua) [reply src/dev/imports.ts]: Yep. Fixed!

**Review by kt3k — APPROVED** [review]
LGTM!

**Pull request merged**